department-of-veterans-affairs/va.gov-cms
# VACMS-11404: Add flagging/unflagging perms for content admin

## Description

Relates to #11285, #11335. Adds permissions so that content admin user role can remove flags for new, deleted, changed filename, changed title on va forms.

## Testing done
1. Made sure content admin can remove flags for forms marked as new or deleted, both
2. Made sure content admin can remove flag for forms marked as having a changed title.
3. Made sure content admin can remove flag for forms marked as having a changed filename. 
4. Behat and other automated tests passing (TBD).

## Screenshots
<img width="1013" alt="Screen Shot 2022-11-07 at 1 59 05 PM" src="https://user-images.githubusercontent.com/22764938/200424302-0a33fb4f-8efe-4672-a423-da891737f2bc.png">


## QA steps

1. Sign in as a user with the content admin role. (Go to people and filter by role > "content admin" to find a user with this role and masquerade as that user.)
5. - [ ] Make sure content admin can remove flags for forms marked as new or deleted, both. (Go to https://cms-rm5itkhhchmq5jzjgvxixgidl6mu241a.ci.cms.va.gov//admin/content/va-forms/ and click on the *new/deleted* tab. Select a form and use the views bulk operations actions menu to remove the flag.)
6. - [ ] Make sure content admin can remove flag for forms marked as having a changed title. (Click on the *changed title* tab. Select a form and use the views bulk operations actions menu to remove the flag.
7. - [ ] Make sure content admin can remove flag for forms marked as having a changed filename. (Click on the *changed filename* tab. Select a form and use the views bulk operations actions menu to remove the flag.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
